index.bs

```diff
@@ -6608,6 +6617,24 @@ At this time, one [=credential property=] is defined: the [=resident key credent
             If {{rk}} is not present, it is not known whether the credential is a [=discoverable credential=] or a [=server-side credential=].
 
             Note: some [=authenticators=] create [=discoverable credentials=] even when not requested by the [=client platform=]. Because of this, [=client platforms=] may be forced to omit the {{rk}} property because they lack the assurance to be able to set it to [FALSE]. [=[RPS]=] should assume that, if the `credProps` extension is supported, then [=client platforms=] will endeavour to populate the {{rk}} property. Therefore a missing {{rk}} indicates that the created credential is most likely a [=non-discoverable credential=].
+
+        :   <dfn>authenticatorDisplayName</dfn>
+        ::  This OPTIONAL property is a [=human palatability|human-palatable=] description of the credential's [=managing authenticator=],
+            chosen by the user.
+
+            The [=client=] MUST allow the user to choose this value,
+            MAY or MAY not present that choice during [=registration ceremonies=],
+            and MAY reuse the same value for multiple credentials with the same [=managing authenticator=] across multiple [=[RPS]=].
+
+            The [=client=] MAY query the [=authenticator=], by some unspecified mechanism, for this value.
+            The [=authenticator=] MAY allow the user to configure the response to such a query.
+            The [=authenticator=] vendor MAY provide a default response to such a query.
+            The [=client=] MAY consider a user-configured response chosen by the user,
+            and SHOULD allow the user to modify a vendor-provided default response.
```

Comment:
I'm wondering how these lines here compliment the three lines above:

```
The [=client=] MUST allow the user to choose this value,
MAY or MAY not present that choice during [=registration ceremonies=],
and MAY reuse the same value for multiple credentials with the same [=managing authenticator=] across multiple [=[RPS]=].
```

Is the idea to push browsers to implementing UI mid-registration that allows the user to confirm or change an autofill'd input box with a value the authenticator or provider provides? But then there's the aspect of a user customizing the value, and the client needing to remember the change for a given provider across multiple sites...it's hard to understand where the source of truth would be.

Reply:
The idea was to give browsers the option, but not obligation, to implement mid-registration UI for this. I tried to write this in a way that an input box somewhere in browser (or password manager) settings would qualify as "allowing the user to choose this value", and the browser could simply omit the value if that setting is not configured.

The above applies mostly to platform authenticators. For external authenticators, the value should presumably be stored on the authenticator and made available to clients by some standardized mechanism such as `authenticatorGetInfo()` in CTAP. In this case, "allow the user to choose" could consist of offering a way to configure the value stored on the authenticator when possible, again with the option but not obligation to do so during the registration.